lib/ansible/modules/database/postgresql/postgresql_sequence.py

```diff
@@ -0,0 +1,646 @@
+        increment=dict(type='int'),
+        minvalue=dict(type='int', aliases=['min']),
+        maxvalue=dict(type='int', aliases=['max']),
+        start=dict(type='int'),
+        cache=dict(type='int'),
+        cycle=dict(type='bool'),
+        schema=dict(type='str', default='public'),
+        cascade=dict(type='bool'),
+        rename_to=dict(type='str'),
+        owner=dict(type='str'),
+        newschema=dict(type='str'),
+        db=dict(type='str', default='', aliases=['login_db', 'database']),
+        session_role=dict(type='str'),
+    )
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        mutually_exclusive=[
+            ['absent', 'data_type'],
+            ['absent', 'increment'],
+            ['absent', 'minvalue'],
+            ['absent', 'maxvalue'],
+            ['absent', 'start'],
+            ['absent', 'cache'],
+            ['absent', 'cycle'],
+            ['absent', 'rename_to'],
+            ['absent', 'owner'],
+            ['absent', 'newschema'],
+            ['rename_to', 'data_type'],
+            ['rename_to', 'increment'],
+            ['rename_to', 'minvalue'],
+            ['rename_to', 'maxvalue'],
+            ['rename_to', 'start'],
+            ['rename_to', 'cache'],
+            ['rename_to', 'cycle'],
+            ['rename_to', 'cascade'],
+            ['rename_to', 'owner'],
+            ['rename_to', 'newschema'],
+            ['cascade', 'data_type'],
+            ['cascade', 'increment'],
+            ['cascade', 'minvalue'],
+            ['cascade', 'maxvalue'],
+            ['cascade', 'start'],
+            ['cascade', 'cache'],
+            ['cascade', 'cycle'],
+            ['cascade', 'owner'],
+            ['cascade', 'newschema'],
+        ]
+    )
+
+    # Note: we don't need to check mutually exclusive params here, because they are
+    # checked automatically by AnsibleModule (mutually_exclusive=[] list above).
+
+    # sequence = module.params["sequence"]
+    # state = module.params["state"]
+    # data_type = module.params["data_type"]
+    # increment = module.params["increment"]
+    # minvalue = module.params["minvalue"]
+    # maxvalue = module.params["maxvalue"]
+    # start = module.params["start"]
+    # cache = module.params["cache"]
+    # cycle = module.params["cycle"]
+    # cascade = module.params["cascade"]
+    # rename_to = module.params["rename_to"]
+    # schema = module.params["schema"]
+    # owner = module.params["owner"]
+    # newschema = module.params["newschema"]
+
+    # Connect to DB and make cursor object:
+    db_connection = connect_to_db(module, autocommit=True)
+    # Change autocommit to False if check_mode:
+    if module.check_mode:
+        db_connection.set_session(autocommit=False)
+    cursor = db_connection.cursor(cursor_factory=DictCursor)
+
+    ##############
+    # Create the object and do main job:
+    data = Sequence(module, cursor)
```

Comment:
```suggestion
    sequence = Sequence(module, cursor)
```
One cosmetic suggestion. I used the name data for the object in postgresql_copy because of it works with data. You can leave it as it is if you like

Reply:
Yes you are right, I liked a shorter name and `sequence` is already used many times in the code, so it is also confusing. I just leave it so :)
Thanks for the review!

Reply:
Nps:)